client/my-sites/stats/hooks/use-is-advanced-feature-enabled.ts

```diff
@@ -1,41 +1,25 @@
-import config from '@automattic/calypso-config';
-import { useNoticeVisibilityQuery } from 'calypso/my-sites/stats/hooks/use-notice-visibility-query';
 import { useSelector } from 'calypso/state';
 import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import useStatsPurchases from '../hooks/use-stats-purchases';
 
 const useIsAdvancedFeatureEnabled = ( siteId: number ) => {
-	const { isCommercialOwned, hasLoadedSitePurchases, isRequestingSitePurchases } =
+	const { supportCommercialUse, hasLoadedSitePurchases, isRequestingSitePurchases } =
 		useStatsPurchases( siteId );
 
 	const isSiteJetpackNotAtomic = useSelector( ( state ) =>
 		isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
 	);
 
-	const canUserManageOptions = useSelector( ( state ) =>
-		canCurrentUser( state, siteId, 'manage_options' )
-	);
 	const canUserViewStats = useSelector( ( state ) =>
 		canCurrentUser( state, siteId, 'view_stats' )
 	);
 
-	const { isLoading: isLoadingNotices, data: purchaseNotPostponed } = useNoticeVisibilityQuery(
-		siteId,
-		'focus_jetpack_purchase',
-		canUserManageOptions
-	);
-
-	const isLoading = ! hasLoadedSitePurchases || isRequestingSitePurchases || isLoadingNotices;
-	const hasAdvancedPlan = isCommercialOwned;
+	const isLoading = ! hasLoadedSitePurchases || isRequestingSitePurchases;
 
-	// to redirect the user can't have a plan purached and can't have the flag true, if either is true the user either has a plan or is postponing
+	// check if the user and the blog should have the access to advanced features
 	const isAdvancedFeatureEnabled =
```

Comment:
For wpcom sites, I think, if they choose to purchase a Jetpack Stats plan, then we should allow them to use the advance features if that makes sense. In that case,  `supportCommercialUse` and `isAdvancedFeatureEnabled` would be identical, meaning we wouldn't need `useIsAdvancedFeatureEnabled` at the stage.

In the future, if the meaning of commercial and advanced feature would diverge, we then could come back and add it.

I went ahead and tweaked it. Hope you don't mind 😄